tensorflow_federated/python/research/robust_aggregation/robust_aggregation_test.py

```diff
@@ -0,0 +1,228 @@
+# Lint as: python3
+# Copyright 2019, Krishna Pillutla and Sham M. Kakade and Zaid Harchaoui.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test Robust Aggregation with Numpy Implementation."""
+
+
+import collections
+import numpy as np
+from six.moves import range
+import tensorflow as tf
+from tensorflow.python.keras.optimizer_v2 import gradient_descent
```

Comment:
In TF >= 1.13 this should be the same as tf.keras.optimizers.SGD(); and so this additional import shouldn't be needed. Could you try using that?